libats-messaging/src/main/scala/com/advancedtelematic/libats/messaging/daemon/MessageBusListenerActor.scala

```diff
@@ -32,9 +36,11 @@ class MessageBusListenerActor[M](source: Source[M, NotUsed])(implicit messageLik
         log.warning("Listener already subscribed. Ignoring Subscribe message")
       case Failure(ex) =>
         log.error(ex, "Source/Listener died, subscribing again")
+        monitor.onError(ex)
         trySubscribeDelayed()
```

Comment:
Probably Instead of handling re-subscription inside of the actor it would be better to stop the actor and use `akka.pattern.BackoffSupervisor` to restart it?